source/directives/checkbox-field.ts

```diff
@@ -49,6 +49,8 @@ module formFor {
      */
     model:BindableFieldWrapper;
 
+    attributes:any;
```

Comment:
Minor: Add a small comment above similar to [the one in `text-field`](https://github.com/bvaughn/angular-form-for/blob/master/source/directives/text-field.ts#L20)